<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetTableClass</class>
 <widget class="QWidget" name="widgetTableClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widgetTable</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="logTabWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="tableBack">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comTable">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>全部项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>检测项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>定标项目</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tableUp">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tableNext">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tableJump">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>跳转</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tableRemove">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tableRemoveAll">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>全部删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
